packages/storybook-react/src/stories/template/PageStep-2.stories.tsx

```diff
@@ -20,101 +25,241 @@ import {
-        <UtrechtDigidButton>
-          <ButtonLink appearance="primary-action-button">
-            Uitloggen
-            <UtrechtIconArrow />
-          </ButtonLink>
-        </UtrechtDigidButton>
-        <Paragraph>Mevrouw Cindy</Paragraph>
-        <br />
-        <br />
-      </aside>
-      <Heading1>
-        <Paragraph lead>ONLINE LOKET</Paragraph>
-        Een verhuizing doorgeven
-      </Heading1>
-      <Heading2>De Verhuizing</Heading2>
-      <Separator />
-      <section>
-        <form>
-          <FormField>
-            <FormLabel htmlFor="verhuisdatum">Verhuisdatum*</FormLabel>
-            <br />
-            <Textbox id="verhuisdatum" type="date" required />
-          </FormField>
-          <Fieldset>
-            <FieldsetLegend>NIEUWE ADRES</FieldsetLegend>
-            <div role="radiogroup" aria-labelledby="nieuwe-adres-label" data-rating-value="">
-              <Heading4 id="nieuwe-adres-label">Hoe wilt u zoeken *</Heading4>
-              <FormField>
-                <RadioButton id="postcode" name="woonplaats-en-straat" />
-                <FormLabel htmlFor="postcode">Postcode</FormLabel>
-              </FormField>
-              <FormField>
-                <RadioButton id="woonplaats-en-straat" name="woonplaats-en-straat" />
-                <FormLabel htmlFor="woonplaats-en-straat">Woonplaats en Straat</FormLabel>
-              </FormField>
+    const onNewAddressSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+      event.preventDefault();
+
+      if (dateRef.current && dateRef.current.value === '') {
+        setInvalid(true);
+        dateRef.current.setAttribute('aria-invalid', 'true');
+      }
+      if (postcodeRef.current && postcodeRef.current.value === '') {
+        console.log(postcodeRef.current.value, 'test');
+        setInvalid(true);
+        postcodeRef.current.setAttribute('aria-invalid', 'true');
+      }
+      if (homeNumberRef.current && homeNumberRef.current.value === '') {
+        setInvalid(true);
+        homeNumberRef.current.setAttribute('aria-invalid', 'true');
+      }
+    };
+
+    const validateField = (current: HTMLInputElement | null) => {
+      if (current && current.value) {
+        if (current.getAttribute('aria-invalid') && current.validity.valid) {
+          current.removeAttribute('aria-invalid');
+          setInvalid(false);
+        } else if (current && current.value === '' && !current.validity.valid) {
+          current.setAttribute('aria-invalid', 'true');
+          setInvalid(true);
+        }
+      }
+    };
+
+    const onNewAddressChange = () => {
+      validateField(dateRef.current);
+      validateField(postcodeRef.current);
+      validateField(homeNumberRef.current);
+    };
+
+    return (
+      <Page {...args} style={pageStyle}>
+        <PageHeader>
+          <Link href="/" aria-label="Gemeente Utrecht">
+            <UtrechtLogo />
+          </Link>
+        </PageHeader>
+        <PageContent role="main">
```

Comment:
Breadcrumbs should not be part of the _main_ landmark. I think that's why we once made `PageContentMain`:

```jsx
<PageContent>
  <BreadcrumbNav/>
  <PageContentMain>...</PageContentMain>
</PageContent>
```

Reply:
Ik ga ook `<PageContentMain rol="main"/>` in plaats van  `PageContentMain` gebruiken want ` PageContentMain` heeft probleem met import